### PR TITLE
Gpc 751 adding gt pv2 messages 97 98

### PR DESCRIPTION
**Pull request by AlexanderKudzevich**

### CODE REVIEW

File: gtpv2/message/message.go [hunk 1]
```diff
@@ -337,6 +337,10 @@ func Parse(b []byte) (Message, error) {
 		m = &ModifyBearerRequest{}
 	case MsgTypeModifyBearerResponse:
 		m = &ModifyBearerResponse{}
+	case MsgTypeUpdateBearerRequest:
+		m = &UpdateBearerRequest{}
+	case MsgTypeUpdateBearerResponse:
+		m = &UpdateBearerResponse{}
```
Comment (dzvancuks) [inline gtpv2/message/message.go:343]: do add _test.go files with UT

Comment (AlexanderKudzevich) [reply gtpv2/message/message.go:343]: Yeah, sure. UT is not ready yet, I need some UpdateBearer examples with bytes for it.

Or I can do it with header only

**Review by dzvancuks — APPROVED** [review]

File: gtpv2/message/update-bearer-req.go [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+			MsgTypeUpdateBearerRequest, teid, seq, nil,
+		),
+	}
+
+	for _, i := range ies {
+		if i == nil {
+			continue
+		}
+		switch i.Type {
+		case ie.BearerContext:
```
Comment (dzvancuks) [inline gtpv2/message/update-bearer-req.go:49]: Could you double check by specks what IE are actually supported by UpdateBearer. Probably not all of these are needed

Comment (AlexanderKudzevich) [reply gtpv2/message/update-bearer-req.go:49]: All IE's are from specification one by one.

File: gtpv2/message/update-bearer-res_test.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+func TestUpdateBearerResponse(t *testing.T) {
+	cases := []testutils.TestCase{
+		{
+			Description: "Normal",
+			Structured: message.NewUpdateBearerResponse(
+				testutils.TestBearerInfo.TEID, testutils.TestBearerInfo.Seq,
+			),
+			Serialized: []byte{
+				// Header
```
Comment (gordeyg) [inline gtpv2/message/update-bearer-res_test.go:30]: Only header? What about other fields?

Comment (AlexanderKudzevich) [reply gtpv2/message/update-bearer-res_test.go:30]: If we look at other's UT, there are some places where the only Header is tested but in messages are a lot of IE's. So as I understand, we can do it later or on PR into the main repo if needed.

Also I need some examples of live traffic with UpdateBearer, but I can't find them.

Comment (gordeyg) [reply gtpv2/message/update-bearer-res_test.go:30]: Don't think it is a good move to take the worst case as reference point. 
You could get these bytes on your own by serializing struct, ensure they are fine and cement with UT by extending Serialized here.

**Review by oicnysa — APPROVED** [review]

**Review by gordeyg — APPROVED** [review]

File: gtpv2/message/update-bearer-res_test.go (hunk 3 above)
Comment (AlexanderKudzevich) [reply gtpv2/message/update-bearer-res_test.go:30]: Done in https://github.com/1NCE-GmbH/go-gtp/pull/9/commits/bbd2d9396dac2c1dd275c9e01b38792a88ca4a22

**Pull request merged**